### PR TITLE
Ice cream reducer

### PR DESCRIPTION
**Pull request by meru-a**
Solves #51

### CODE REVIEW

File: client/src/App.tsx
Comment (larkinds) [inline client/src/App.tsx:1]: Reminder to keep App clean! Perhaps move this to some sort of example file?

**Review by larkinds — CHANGES_REQUESTED** [review]
This looks great, and it works! Good work. Just one small change - the example of the working reducer is in App.tsx

File: client/src/App.tsx
Comment (meru-a) [reply client/src/App.tsx:1]: Cleaned the app.tsx!

**Review by larkinds — APPROVED** [review]
Sweet, thanks!

**Pull request merged**